README.md

```diff
@@ -85,4 +85,5 @@ In order to upgrade the Subnet-EVM version, you need to change the version in `g
 [v0.2.4] AvalancheGo@v1.11.11-v1.11.12 (Protocol Version: 37)
 [v0.2.5] AvalancheGo@v1.11.13-v1.12.1 (Protocol Version: 38)
 [v0.3.0] AvalancheGo@v1.12.2 (Protocol Version: 39)
+[v0.4.0] AvalancheGo@v1.12.2/1.13.0-fuji/1.13.0 (Protocol Version: 39)
```

Comment:
this is not introducing a breaking change, we should update it to v0.3.1 (as we did in subnet-evm)